owner/manager_test.go

```diff
@@ -204,39 +173,27 @@ func TestCluster(t *testing.T) {
 		WithInfoCache(ic3),
 	)
 	err = d3.Start(nil)
-	if err != nil {
-		t.Fatalf("DDL start failed %v", err)
-	}
+	require.NoErrorf(t, err, "DDL start failed")
 	defer func() {
 		err = d3.Stop()
-		if err != nil {
-			t.Fatal(err, IsNil)
-		}
+		require.NoErrorf(t, err, "DDL stop failed")
 	}()
 	isOwner = checkOwner(d3, false)
-	if isOwner {
-		t.Fatalf("expect false, got isOwner:%v", isOwner)
-	}
+	require.Falsef(t, isOwner, "expect false, got isOwner true")
 	err = d3.Stop()
-	if err != nil {
-		t.Fatal(err, IsNil)
-	}
+	require.NoErrorf(t, err, "DDL stop failed")
 
 	// Cancel the owner context, there is no owner.
 	err = d1.Stop()
-	if err != nil {
-		t.Fatal(err, IsNil)
-	}
+	require.NoErrorf(t, err, "DDL stop failed")
 	session, err := concurrency.NewSession(cliRW)
-	if err != nil {
-		t.Fatalf("new session failed %v", err)
-	}
+	require.NoErrorf(t, err, "new session failed")
 	elec := concurrency.NewElection(session, DDLOwnerKey)
 	logPrefix := fmt.Sprintf("[ddl] %s ownerManager %s", DDLOwnerKey, "useless id")
 	logCtx := logutil.WithKeyValue(context.Background(), "owner info", logPrefix)
 	_, err = owner.GetOwnerInfo(goctx.Background(), logCtx, elec, "useless id")
 	if !terror.ErrorEqual(err, concurrency.ErrElectionNoLeader) {
-		t.Fatalf("get owner info result don't match, err %v", err)
+		require.NoErrorf(t, err, "get owner info result don't match, err %v")
```

Comment:
Could be `require.Truef(t, terror.ErrorEqual(err, concurrency.ErrElectionNoLeader), "get owner info result don't match, err %v"))`.

Reply:
Won't this be too long and affect readability?

Reply:
I think some editors will wrap the line automatically. The thing is that the change here is not equivalent. `NoError` checks if it is nil, while the origin branch checks if it is a specific error.